src/main/java/frc/robot/DriveBase.java

```diff
@@ -48,12 +50,19 @@
     //Stores encoder position
     private double initEncLeft, initEncRight, initEncCenter;
 
+    //joystick ports
```

Comment:
Class variables should be private, not public (also please remove these since we're not using Joysticks in DriveBase).